src/schemas/json/package.json

```diff
@@ -76,34 +76,34 @@
       "type": "string"
     },
     "packageExportsEntryPath": {
-      "type": "string",
-      "description": "The module path that is resolved when this specifier is imported.",
+      "type": ["string", "null"],
+      "description": "The module path that is resolved when this specifier is imported. Set to `null` to disallow importing this module.",
```

Comment:
I'm not doubting you here at all, just curious if you can point me to some documentation or anything that says you can use `null` here? I thought all you'd have to do is just omit the module from the object...

The docs page you linked does not mention `null` at all.